src/lib/y2storage/encryption_processes/base.rb

```diff
@@ -0,0 +1,69 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+
+module Y2Storage
+  module EncryptionProcesses
+    # Base class to perform the creation of an encrypted device
+    #
+    # @note This is a base class. To really created an encrypted device use any
+    # of its subclasses, which must define an #enccryption_type
```

Comment:
Typo: `#enccryption_type`

Reply:
I would comment that `EncryptionProcesses` are intended to be used through a `EncryptionMethod`.